.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Snuba Tests
+on:
+  push:
+    branches:
+      - master
+      - releases/**
+  pull_request:
+
+jobs:
+  tests:
+    name: Snuba tests
+    runs-on: ubuntu-18.04
```

Comment:
Why not latest?

Reply:
> Why not latest?

The only downside is when Github swaps the OS version under us it *might* make the CI fail.

Now, since we actually run all the steps within docker I assume the likelihood of something breaking can be reduced, thus, I'm comfortable to use `ubuntu-latest` and have the chance of being surprised later on.